ReportService/ReportTask/ReportTask.cs

```diff
@@ -21,67 +21,56 @@ public partial class ReportTask : IReportTask
         public string Name { get; }
         public DtoSchedule Schedule { get; }
         public DateTime LastTime { get; private set; }
-        public List<IOperation> Operations { get; set; }
-        public Dictionary<string, object> Parameters { get; set; } //todo: change to class?
-        public List<ParameterInfo> ParameterInfos { get; set; }
-        public List<TaskDependency> DependsOn { get; set; }
+        public List<IOperation> Operations { get; set; } = new List<IOperation>();
+        public Dictionary<string, object> Parameters { get; set; } = new Dictionary<string, object>();//todo: change to class?
+        public List<ParameterInfo> ParameterInfos { get; set; } = new List<ParameterInfo>();
+        public List<TaskDependency> DependsOn { get; set; } = new List<TaskDependency>();
 
-        private readonly IMonik monik;
-        private readonly ILifetimeScope autofac;
```

Comment:
is scope or container more precise naming?

Reply:
Makes sense.